client/my-sites/plugins/search-box-header/index.jsx

```diff
@@ -20,6 +20,7 @@ const SearchBox = ( { isMobile, searchTerm, doSearch, searchBoxRef, isSearching
 				pinned={ isMobile }
 				fitsContainer={ isMobile }
 				onSearch={ doSearch }
+				onSearchClose={ () => setQueryArgs( {} ) }
```

Comment:
This creates a second `page()` call. `onSearch('')` is [already being called](https://github.com/Automattic/wp-calypso/blob/d2e27c7b2f8120cdd7861fe37bacc41921428f97/packages/search/src/search.tsx#L240) on close(clicking the "X").

```suggestion
```